### DISCUSSION TITLE
Static DNS entries

### DISCUSSION THREAD

**Discussion by santa-collab** [Ylianst/MeshCentral#4549] [Q&A]
We are initiating the process of decommissioning an older server, due to hardware and OS related, with an updated replacement. The server acts as DNS and there may be servers/services within the organization that point to this server either statically or dynamic under the TCP/IP network options. 

These other servers contain Mesh already and I noticed Mesh currently collects the Ethernet, IPv4, IPv6, and MAC Layer, but nothing related to DNS. Is this something where we can configure the config.json to populate this information?

I reviewed https://meshcentral-community.com/doku.php?id=reference:config but it appears to not include an option for this. Thanks!

**Comment by silversword411** [comment]
I don't see anything in the config options...but you could use Terminal and type `ipconfig /all` :)

**Comment by santa-collab** [comment]
This doesn't seem to work as the information shows up as truncated. Also to perform this on the multiples of servers would take a bit of time, if it worked. 

I created a simple script via PowerShell, but it does not appear to want to perform the command when pushed out to the group or an individual machine. 

`#Get name of workstation
$h = hostname

#The Get-DnsClientServerAddress cmdlet gets one or more DNS server IP addresses associated with the interfaces on the computer.

Get-DnsClientServerAddress -AddressFamily IPv4 | Select InterfaceAlias, ServerAddresses | Out-File "C:\Temp\DnsClientSrvAddr_$h.csv" 

#Move file to new destination (ie. shared drive)

Move-Item -Path "C:\Temp\DnsClientSrvAddr_$h.csv" -Destination "\\REMOTE_DESTINATION_FILESHARE"`
